File: src/AxisCRM.Api/Domain/Models/Parecer.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace AxisCRM.Api.Domain.Models
{
    public class Parecer : EntidadeBase
    {
        public int IdAtendimento { get; set; }
        public int IdUsuario { get; set; }
        public string Descricao { get; set; }
        public string PessoaContato { get; set; }
        public DateTime? DataUltimaAlteracao { get; set; }

        [ForeignKey(nameof(IdAtendimento))]
        public virtual Atendimento Atendimento { get; set; }
        
        [ForeignKey(nameof(IdUsuario))]
        public Usuario Usuario { get; set; }
    }
}

File: App/src/models/Parecer.js
export default class Parecer {
  constructor({
    id = 0,
    descricao = '',
    dataCadastro = null,
    pessoaContato = '',
    status = 0,
    idAtendimento = 0,
    idUsuario = 0,
    dataUltimaAlteracao = null
  } = {}) {
    this.id = id;
    this.descricao = descricao;
    this.dataCadastro = dataCadastro
      ? new Date(dataCadastro)
      : null;
    this.pessoaContato = pessoaContato;
    this.status = status;
    this.idAtendimento = idAtendimento;
    this.idUsuario = idUsuario;
    this.dataUltimaAlteracao = dataUltimaAlteracao
      ? new Date(dataUltimaAlteracao)
      : null;
  }
}
